src/transformers/models/llama/convert_llama_weights_to_hf.py

```diff
@@ -231,10 +240,16 @@ def permute(w):
     shutil.rmtree(tmp_model_path)
 
 
-def write_tokenizer(tokenizer_path, input_tokenizer_path):
+def write_tokenizer(tokenizer_path, input_tokenizer_path, use_fast=True):
```

Comment:
```suggestion
def write_tokenizer(tokenizer_path, input_tokenizer_path):
```
Let's just do both in the conversion script. It should create a complete repo.